packages/observability/__tests__/client/providers/sentry-client.test.ts

```
@@ -0,0 +1,634 @@
+      // Mock window to throw an error when accessed
+      Object.defineProperty(globalThis, 'window', {
+        get() {
+          throw new Error('Mock initialization error');
+        },
+        configurable: true,
+      });
+
+      await expect(provider.initialize(config)).rejects.toThrow('Failed to initialize Sentry: Error: Mock initialization error');
+
+      // Restore
+      delete (globalThis as any).window;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)